### PR TITLE
Po switch list button

### LINKED ISSUE #41: conditional home page that displays text that tells you you are already on a list and a "change list" button

### PR DESCRIPTION
**Pull request by patty-rose**
## Description
This PR adds conditional rendering to the home page so it will show that you are on a list if you are already on a list when you arrive. And then hides the switch list form in a button that can toggle. It also edits the handleNewToken function so that it creates a new token when you click it on the Home page.

## Related Issue

closes #41 

### Before
Before you did not know if you were already on a list on the home page. And the create new list button didn't create a new list if you were already on a list.
https://github.com/the-collab-lab/tcl-65-smart-shopping-list/assets/101524326/a158de58-b260-4602-a157-31535cdcdbac

## Testing Steps / QA Criteria

this is a quick and dirty PR-- my apologies! 
Play around in the GH preview on the home page and see how the button, form, and currentList displays toggle.

### CODE REVIEW

**Review by shannonerhea — APPROVED** [review]
Excellent upgrade. Helpful to User.

File: src/App.jsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ export function App() {
 	);
 
 	const handleNewToken = () => {
-		//if token is true do nothing
-		if (listToken) return;
-		//if token is false setlistToken calls the generateToken function and sets the result to state var of listToken
+		// //if token is true do nothing
+		// if (listToken) return;
+		// //if token is false setlistToken calls the generateToken function and sets the result to state var of listToken
```
Comment (lydiadunning) [inline src/App.jsx]: These comments are no longer accurate.

**Review by lydiadunning — APPROVED** [review]
Looks great!  Putting the forms into variables makes this page easier to read, too.

**Review by isaiahasanchez — APPROVED** [review]
Great change! Code looks good, I agree with what the others said as well.

**Pull request merged**